packages/server-wallet/src/__test-with-peers__/create-and-close-channel/ledger-funding.test.ts

```diff
@@ -24,7 +24,13 @@ import {
 
 const ETH_ASSET_HOLDER_ADDRESS = makeAddress(ethers.constants.AddressZero);
 
-const tablesUsingLedgerChannels = ['channels', 'ledger_requests', 'ledger_proposals'];
+const tablesUsingLedgerChannels = [
+  'channels',
+  'ledger_requests',
+  'ledger_proposals',
```

Comment:
Is the `LedgerProposals` table/model unused now? If so, should we remove code that references the model and write a db migration to remove the table?

Reply:
We can take care of this after this PR is merged. Filed https://github.com/statechannels/statechannels/issues/3371, so that we don't forget.